### PR TITLE
connection property should be "user-agent".

### PR DESCRIPTION
**Pull request by amarzavery**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,64 +81,169 @@ You can run the examples by cloning the repo or copy pasting the below sample in
...
+ *         - `"sb://<your-namespace>.servicebus.windows.net/<event-hub-name>/ConsumerGroups/<consumer-group-name>/Partitions/<partition-id>"`.
+ *
+ *     - **ManagementClient**
+ *         - `"sb://<your-namespace>.servicebus.windows.net/<event-hub-name>/$management"`.
+ */
+export async function authenticate(audience, closeConnection = false) {
+  await connectionContext.cbsSession.init();
+  const tokenObject = await connectionContext.tokenProvider.getToken(audience);
+  const result = await connectionContext.cbsSession.negotiateClaim(audience, tokenObject);
+  console.log("Result is: %O", result);
```
Comment (daschult) [inline README.md]: Can you use string interpolation here instead of a formatted string?

File: README.md [hunk 2]
```diff
@@ -78,64 +81,169 @@ You can run the examples by cloning the repo or copy pasting the below sample in
...
+ *          - `"sb://<yournamespace>.servicebus.windows.net/<hubName>"`
+ *          - `"sb://<yournamespace>.servicebus.windows.net/<hubName>/Partitions/<partitionId>"`.
+ *
+ *     - **Receiver**
+ *         - `"sb://<your-namespace>.servicebus.windows.net/<event-hub-name>/ConsumerGroups/<consumer-group-name>/Partitions/<partition-id>"`.
+ *
+ *     - **ManagementClient**
+ *         - `"sb://<your-namespace>.servicebus.windows.net/<event-hub-name>/$management"`.
+ */
+export async function authenticate(audience, closeConnection = false) {
```
Comment (daschult) [inline README.md]: Are there helper functions to assist in creating these audience URL strings?

File: README.md [hunk 3]
```diff
@@ -78,64 +81,169 @@ You can run the examples by cloning the repo or copy pasting the below sample in
 ## Example 1 - CBS Authentication
```
Comment (daschult) [inline README.md]: What does CBS stand for?

File: README.md [hunk 4]
```diff
@@ -78,64 +81,169 @@ You can run the examples by cloning the repo or copy pasting the below sample in
...
```
Comment (daschult) [inline README.md]: You may want to somehow indicate (maybe in the Example 1 header) that this code block is referred to later as `./cbsAuth`.

File: README.md (hunk 2 above)
Comment (amarzavery) [reply README.md]: The audience construction happens in the relevant sdks (@azure/event-hubs) and (@azure/service-bus). The intent of the example is to show how things can be done with the lower level library. However, you have made a very good point. We can definitely add some helper methods to create address and audience for a specific entity (EventHub Sender, EventHub Receiver, ServiceBus QueueSender, etc.) from the provided connection string. These methods could be a part of the connection config.

File: README.md (hunk 3 above)
Comment (amarzavery) [reply README.md]: `Claims Based Service`. Thanks for pointing that 👍

**Pull request merged**